apps/safari/safari.py

```diff
@@ -28,7 +25,8 @@ def browser_open_address_in_new_tab():
 class BrowserActions:
     def address() -> str:
         try:
-            window = safari_app().windows()[0]
+            mac_app = ui.apps(bundle=actions.app.bundle())[0]
+            window = mac_app.windows()[0]
```

Comment:
I think maybe we could just use ui.active_app() here?

Reply:
SGTM. Done. Seems to work with local testing.